project/ScalafixBuild.scala

```diff
@@ -342,4 +342,16 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
       }
     }
   )
+  private def testRulesFor(scalaVersion: String, state: State): State = {
+    supportedScalaVersionforRules(scalaVersion)
+      .filter(_ != scalaVersion)
+      .flatMap { v =>
+        List(
+          s"""set testsInput/scalaVersion := "$v"""",
+          s"""set testsOutput/scalaVersion := "$v"""",
+          s"unit/testOnly scalafix.tests.rule.RuleSuite"
```

Comment:
I am not familiar with the syntax, but can that `RuleSuite` be excluded from the `unit/test` above? (to avoid the current version being tested twice)

Reply:
The current version is not tested twice anymore, since I filter `.filter(_ != scalaVersion) `

Reply:
isn't the current version tested in the `unit/test` that exists on master?

Reply:
the current version is tested in fact in unit/test,
then we set a different version for testsInput that is different from the current version. 

Currently, we don't test at the same place ruleSuite for all supportedVersions..

Reply:
my bad I misread the filter indeed!

Reply:
maybe a clearer implementation would be to rename `latestVersions` to `previousVersions`, to exclude the current version in the implementation